src/controller/CommissioningDelegate.h

```diff
@@ -566,6 +608,13 @@ class CommissioningParameters
     Optional<bool> mAttemptWiFiNetworkScan;
     Optional<bool> mAttemptThreadNetworkScan; // This automatically gets set to false when a ThreadOperationalDataset is set
     Optional<bool> mSkipCommissioningComplete;
+
+    Optional<NodeId> mICDControllerNodeId;
+    Optional<uint64_t> mICDSubjectId;
+    Optional<ICDRegistrationDelegate::ICDKey> mICDSymmetricKey;
+    Optional<ICDRegistrationDelegate::ICDKey> mICDVerificationKey;
```

Comment:
This isn't necessary for a commissioner.

Reply:
updated